### DISCUSSION TITLE
What are the known problems with Sanoid and daylight savings time?

### DISCUSSION THREAD

**Discussion by parke** [jimsalterjrs/sanoid#747] [Q&A] [answered]
From [README.md](https://github.com/jimsalterjrs/sanoid): 

```
A typical Sanoid system would have a single cron job but see INSTALL.md fore more details:

* * * * * TZ=UTC /usr/local/bin/sanoid --cron

Note: Using UTC as timezone is recommend to prevent problems with daylight saving times
```

What are the known problems with using DST and Sanoid?

Any known problems with DST and `syncoid`?

**Comment by redmop** [comment]
When time changes happen, there can be snapshot name collisions. UTC doesn't have that problem.

**Reply by parke** [reply]
Are there any serious consequences of such collisions?

Will snapshots resume automatically after we exit the 60-minute collision window?

**Comment by redmop** [comment, marked as answer]
I've not had any issues. I use localtime, which has time changes. I lose snapshots that would happen when the time change happens, or skip one.

**Comment by jimsalterjrs** [comment]
There are no serious repercussions generally, because Sanoid was designed to tolerate conditions up to and including "laptop" in terms of messiness and availability. The worst thing that may happen is you miss taking a single hourly when you "fall back," because it's been the same hour twice in a row.

**Comment by jimsalterjrs** [comment]
Oh, and syncoid absolutely does not care about DST shifts. Syncoid doesn't even care what *time* it is, beyond naming sync snapshots (if you haven't set `--no-sync-snap`).
